js/src/get-started-page/get-started-card/index.js

```diff
@@ -28,66 +27,61 @@ const GetStartedCard = () => {
 	const disableNextStep = ! glaData.mcSupportedLanguage;
 
 	return (
-		<Card className="woocommerce-marketing-google-get-started-card">
-			<Flex>
-				<FlexBlock className="motivation-text">
-					<Text variant="title.medium" className="title">
-						{ __(
-							'List your products on Google, for free and with ads',
+		<Card className="gla-get-started-card" isBorderless>
+			<FlexItem className="motivation-image">
+				<img
+					src={ motivationImageURL }
+					alt={ __(
+						'Google Shopping search results example',
+						'google-listings-and-ads'
+					) }
+					width="279.05"
+					height="185"
+				/>
+			</FlexItem>
+			<CardBody>
+				<Text
+					variant="title.medium"
+					className="gla-get-started-card__title"
+				>
+					{ __(
+						'Get your products in front of more shoppers with Google Listings & Ads',
+						'google-listings-and-ads'
+					) }
+				</Text>
+				<AppButton
+					isPrimary
+					disabled={ disableNextStep }
+					href={ getSetupMCUrl() }
+					eventName="gla_setup_mc"
+					eventProps={ {
+						target: 'set_up_free_listings',
+						trigger: 'click',
+					} }
```

Comment:
Thanks for raising this up @eason9487!

@j111q as Eason's question, we now have two `Start listing products →` CTA in the new get started page, do we see the two CTA clicks are two different events, or were they actually the same? Thanks!

@eason9487

> Otherwise, they may be aggregated into the same event from the tracking data.

I don't quite understand what does this mean. My understanding of events aggregation is if we send the events with the same event name and same props, they would be seen as the same one, otherwise not. So in this case, if the answer from Jill is we don't need to distinguish the two events, we just keep the code as is. But if we need to distinguish them, we should send two different event names from each click. Do I understand it correctly?

Reply:
> My understanding of events aggregation is if we send the events with the same event name and same props, they would be seen as the same one, otherwise not.

I think so. Although the tracking function will capture more data internally, not 100% sure if they could still distinguish by those ones. It would be easier to make them differentiate from the tracking data GLA sent.

> So in this case, if the answer from Jill is we don't need to distinguish the two events, we just keep the code as is.

Yes.

> But if we need to distinguish them, we should send two different event names from each click. Do I understand it correctly?

I think we could add another event property. For instance, [the `gla_chart_tab_click` event](https://github.com/woocommerce/google-listings-and-ads/blob/a819c2ea56bfe9a4b871c0b5f379e4aa560c033b/js/src/reports/summary-section.js#L23-L25). In this way, if an analyst wants to know "What is the difference in conversion rates between two CTA buttons to the onboarding setup?", they can aggregate data into two groups by the same event name and, for example, `context`. And if an analyst wants to know "What the conversion rate from the get started page to the onboarding setup", they can just use the same event name to aggregate data.

Reply:
I updated the code in 5faf81d, I think it's more flexible as we can both see them as the same event or two different events simply filtering by the new property `context`. Thanks @eason9487!

Reply:
Thanks for the change!

To make the tracking doc up-to-date, it would be nice to have the corresponding updates in their event JSDoc.

https://github.com/woocommerce/google-listings-and-ads/blob/5faf81dc65cd2ce9bdb855098905143a5e85ca97/js/src/utils/recordEvent.js#L160-L166
https://github.com/woocommerce/google-listings-and-ads/blob/5faf81dc65cd2ce9bdb855098905143a5e85ca97/js/src/get-started-page/get-started-card/index.js#L24
https://github.com/woocommerce/google-listings-and-ads/blob/5faf81dc65cd2ce9bdb855098905143a5e85ca97/js/src/get-started-page/get-started-with-video-card/index.js#L27

Reply:
Ah, miss the event JSDoc again. Added in d5cf7d3 and noted, thanks!